lib/internal/domexception.js

```diff
@@ -43,6 +43,12 @@ class DOMException extends Error {
   }
 }
 
+Object.defineProperties(DOMException.prototype, {
+  name: { enumerable: true, configurable: true },
```

Comment:
Nit: the `configurable: true` isn't necessary, as the property is already configurable and Object.defineProperties won't override it.

Reply:
I'd prefer to be explicit here since `Object.defineProperties` will read inherited properties of descriptors and Node internals aren't run in an isolated realm yet.

Reply:
`DOMException` was created right above this with its prototype having them as own properties, so that is not an issue here.